src/components/POLine/POLineForm.js

```diff
@@ -447,6 +450,17 @@ function POLineForm({
                             integrationConfigs={integrationConfigs}
                           />
                         </Accordion>
+                        <Accordion
+                          id={ACCORDION_ID.donorsInformation}
+                          label={<FormattedMessage id="ui-orders.line.accordion.donorInformation" />}
+                        >
+                          <DonorsList
+                            name="donorOrganizationIds"
+                            donorOrganizationIds={donorOrganizationIds}
+                            visibleColumns={['name', 'code', 'unassignDonor']}
```

Comment:
should be specified inside the component, that is responsible for donor manipulation in the form